Dockerfile

```diff
@@ -1,18 +1,29 @@
+ARG EXECUTABLE_NAME=mysql-resources-db-go-service
+
 FROM golang:1.15.2-alpine
 
-WORKDIR $GOPATH/src/mysql-resources-db-go-service
+ENV ROOT_PACKAGE=github.com/artofimagination/$EXECUTABLE_NAME
```

Comment:
Basicly, you could use a docker image for build your code and move the compiled executor to a smaller image for execution. With this you could reduce the size of your container image.

Reply:
Do you mean, that possibly I can include in the future an old library that could not work, because I am using a path without the "github.com/..." in it?